docs/devenv_rhel8.md

```diff
@@ -96,22 +111,44 @@ Download the OpenShift pull secret from the https://console.redhat.com/openshift
 Run the following commands to configure CRI-O for using the pull secret when fetching container images.
 ```bash
 sudo chmod 600 /etc/crio/openshift-pull-secret
-sudo mkdir -p /etc/crio/crio.conf.d/
-sudo cp ~microshift/microshift/packaging/crio.conf.d/microshift.conf /etc/crio/crio.conf.d/
-sudo systemctl restart crio && sleep 3
-echo 1 | ~microshift/microshift/hack/cleanup.sh
 ```
 
-### Running MicroShift
-Run the MicroShift in the background using the following command.
+### Configuring MicroShift
+MicroShift requires system configuration updates before it can be run. These updates include `CRI-O`, networking and file system customizations.
+
+It is necessary to install the MicroShift RPM packages to automatically apply all the necessary customizations.
+```
+sudo subscription-manager repos --enable fast-datapath-for-rhel-8-$(uname -i)-rpms
+sudo dnf install -y ~/microshift/packaging/rpm/_rpmbuild/RPMS/*/*.rpm
+```
+
+The MicroShift service is not enabled or started by default. Start the service once to trigger the initialization sequence of all the dependencies.
 ```bash
-nohup sudo ./microshift run >> /tmp/microshift.log &
+sudo systemctl start microshift
 ```
-Examine the `/tmp/microshift.log` log file to ensure successful startup.
+
+Wait until all the pods are up and running.
+```bash
+watch sudo $(which oc) --kubeconfig /var/lib/microshift/resources/kubeadmin/kubeconfig get pods -A
+```
+
+Finalize the procedure by stopping the MicroShift service and cleaning up its images and configuration data.
+```
+echo 1 | ~/microshift/hack/cleanup.sh
+```
+
+It should now be possible to run a standalone MicroShift executable file as presented in the next section.
+
+### Running MicroShift Executable
+Run the MicroShift executable file in the background using the following command.
+```bash
+nohup sudo ~/microshift/microshift run >> /tmp/microshift.log &
```

Comment:
We can also suggest to copy to /usr/bin and restart the service as an alternative 

I think your proposal is still useful if you want to debug

Reply:
Done

Reply:
Done